sphinx-doc/network.rst

```diff
@@ -0,0 +1,12 @@
+.. _network:
+
+Network
+=====================
+You can disable or enable network cache using the following commands ::
+  M-x indium-webkit-disable-cache
+  M-x indium-webkit-enable-cache
+
+Both commands save your choice which will be applied on next indium launch for a current session.
```

Comment:
`Both commands save your choice which will be used for future Indium connections for the current Emacs session`?